src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -123,8 +123,9 @@ CLASS zcl_abapgit_objects DEFINITION
         !iv_package TYPE devclass .
     CLASS-METHODS delete_object
       IMPORTING
-        !iv_package TYPE devclass
-        !is_item    TYPE zif_abapgit_definitions=>ty_item
+        !iv_package   TYPE devclass
```

Comment:
wonder if its possible to remove "iv_package" from delete_object, if possible suggest as a follow up change

Reply:
I don't think we can remove `iv_package` because there are a few object types that use `iv_package` to update tadir.